tools/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "editorsyntax",
+  "license" : "MIT",
+  "description": "PowerShell language syntax",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PowerShell/EditorSyntax.git"
+  },
+  "version": "1.0.0",
+  "dependencies": {
+    "fast-plist": "0.1.2"
```

Comment:
I need to register that we are using this library (internal process). I'm expecting it to be approved quickly but I wanted to let you know. It might take a day.

Reply:
If it helps fast-plist was written by Microsoft https://github.com/Microsoft/node-fast-plist

Reply:
lol

Reply:
well that makes things a lot easier